extension/controller/backgroundScript.js

```diff
@@ -163,4 +174,25 @@ const messageListener = async function(message, sender) {
 }
 
 browser.runtime.onMessage.addListener(messageListener);
-browser.experiments.translationbar.onTranslationRequest.addListener(messageListener);
\ No newline at end of file
+browser.experiments.translationbar.onTranslationRequest.addListener(messageListener);
+
+// loads fasttext (languaage detection) wasm module and model
+fetch(browser
```

Comment:
nit: we could extract fasttext loading and detection logic to a separate class to simplify bgScript

Reply:
there's quite enough fasttext classes/artifacts already, adding another one just to load files can make this over-engineered